bosh-shell/startup.sh

```diff
@@ -2,25 +2,25 @@
 
 set -eu
 
-if [ -z "${BOSH_ID_RSA:-}" ] || [ -z "${BOSH_IP:-}" ] || [ -z "${BOSH_ENVIRONMENT:-}" ] || [ -z "${BOSH_CA_CERT:-}" ]; then
-  echo "WARNING: You need to set BOSH_ID_RSA, BOSH_IP, BOSH_ENVIRONMENT and BOSH_CA_CERT."
+if [ -z "${USER_ID_RSA:-}" ] || [ -z "${BOSH_IP:-}" ] || [ -z "${BOSH_ENVIRONMENT:-}" ] || [ -z "${BOSH_CA_CERT:-}" ]; then
+  echo "WARNING: You need to set USER_ID_RSA, BOSH_IP, BOSH_ENVIRONMENT and BOSH_CA_CERT."
   echo "You will not be logged into Bosh."
   exec bash
 fi
 
-echo "$BOSH_ID_RSA" | base64 -d > /tmp/bosh_id_rsa && chmod 400 /tmp/bosh_id_rsa
+echo "$USER_ID_RSA" | base64 -d > /tmp/user_id_rsa && chmod 400 /tmp/user_id_rsa
```

Comment:
Why do we base64 this?

Reply:
Probably `docker run -e USER_ID_RSA=<base64 encoded>`

Reply:
Makes sense. Tested and it works.